### ISSUE TITLE
Regular zarr/n5 community conference calls (Open to all)

### ISSUE THREAD

**Issue by jakirkham** [zarr-developers/community#1]
### tl;dr:
- biweekly calls have been 7-8pm Europe/London time on Wednesdays, beginning in 2020 from January 8
- Zoom link: https://zoom.us/j/300670033?pwd=OFhjV0FHQmhHK2FYbGFRVnBPMVNJdz09 (Passcode: 558943)
- [rolling agenda/minutes doc](https://hackmd.io/tFcffc__QRuo1UdGcGUbsg)

More information as well as a Google calendar ([ics](https://calendar.google.com/calendar/ical/c_ba2k79i3u0lkf49vo0jre27j14%40group.calendar.google.com/public/basic.ics)) are available under https://zarr.dev/#community-meetings

### Original comment
Raised in issue ( https://github.com/zarr-developers/zarr/issues/291 ) was the idea of having regular conference calls to discuss more complex issues in a high bandwidth setting. Opening here to discuss the logistics (e.g. how often, how long, what day, making agendas, what medium, how to join, etc.).

**Comment by alimanfoo** [comment]
On the last call we agreed to revisit the scheduling of the conference calls. So there will be **no conference call today**. @jakirkham kindly agreed to setup a new doodle.

**Comment by jakirkham** [comment]
Sorry for the delay. Have been out with a cold since SciPy. Will add the poll Monday.

**Comment by jakirkham** [comment]
Here's [a poll]( https://www.when2meet.com/?7949264-ncfyx ). I left the times unbounded to avoid timezone bias. Also did this poll for the next 2 weeks given that will be the new frequency.

**Comment by jakirkham** [comment]
cc @llllllllll

**Comment by joshmoore** [comment]
@jakirkham : @WardF and I hung out 2 weeks ago and are in the zoom now if you want to join.  It's pretty last minute though. I'd assume due to the turn out that we either need the poll or at least a definitive statement of the next attempt.

**Comment by joshmoore** [comment]
Next meeting is planned for Oct 23 at the same time (2000 UK). Agenda items can be added to https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit#heading=h.mw5m9r245ow0

**Comment by jakirkham** [comment]
Thanks for doing that Josh! 😄

**Comment by joshmoore** [comment]
Updated my post that was made on the _24th_ of October to say that it's tonight the 5th of Nov. Sheesh.

**Comment by WardF** [comment]
I have a conflict for this meeting, unfortunately.  I will hopefully be on for at least part of it.

**Comment by joshmoore** [comment]
Just a reminder to add anything to the agenda that you'd like to potentially have discussed tomorrow: Nov. 20 (I triple checked  my dates this time!)

**Comment by joshmoore** [comment]
Blank agenda added for tomorrow, Wed. the 4th, if anyone would like to add items.

**Comment by joshmoore** [comment]
And one added for Wed, the 18th, just in case anyone is around. Other than that, 🎄(or similar). I won't be back around until the 8th of January at the earliest.

**Comment by jakirkham** [comment]
Unfortunately I won't be able to make it. Let's try to meet on the 8th in the New Year. Happy Holidays all! 😄

**Comment by ryan-williams** [comment]
seems like folks didn't meet this past Weds 2020-01-22; should we start again next Weds, 2020-01-29?

If I'll make a cal event again if so (that was how I was usually reminded of them previously, thx @joshmoore 😀)

**Comment by jakirkham** [comment]
If you do make an event @ryan-williams, could you please send it out to the group? 🙂

**Comment by ryan-williams** [comment]
(I made the cal event and invited everyone who I had on the previous one)

**Comment by joshmoore** [comment]
This [evening](https://arewemeetingyet.com/UTC/2020-03-25/20:00/b/Zarr%2FN5%20community%20conference%20call) @sofroniew (napari) and @thewtex (ITK) will be joining to discuss spatial dimensions (especially of images) in Zarr and the related ecosystem. Other topics and everyone are welcome.

**Comment by joshmoore** [comment]
It's that time again, this [evening](https://arewemeetingyet.com/UTC/2020-03-25/20:00/b/Zarr%2FN5%20community%20conference%20call) half the agenda is up in the air for community thoughts/comments/use-cases but we'll try to focus at least half of the time on moving the current [governance](https://github.com/zarr-developers/governance/pull/9) push forward. See you there.

**Comment by alimanfoo** [comment]
Thanks @joshmoore, just to say I should be able to make it tonight, look forward to catching up.

**Comment by joshmoore** [comment]
I failed to factor in summer time with the URLs above, so being more careful: tonight is again the regular community call at:

 2100 Berlin
 2000 London
 1500 NYC
 1200 SF
 
See some of you there!

**Comment by alimanfoo** [comment]
Hi all, apologies I can't attend tonight. Please ping me if there's anything I can do to push forward on the numfocus grant (or elsewhere).

**Comment by joshmoore** [comment]
Friendly reminder that [the regular meeting](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit#heading=h.amll2jwn5v1h) is this evening at [the most recently posted](https://github.com/zarr-developers/community/issues/1#issuecomment-617868724) time slots.

**Comment by joshmoore** [comment]
@jakirkham has kindly invited some @NeurodataWithoutBorders ( https://www.nwb.org ) folks for today's call if anyone would like to pop in.

**Comment by Carreau** [comment]
As brought out today in the community call, I'd like to make a smaller meeting limited to discuss the zarr v3 spec. 

@alimanfoo, @martindurant @joshmoore, @jakirkham, @ryan-williams, @andrewfulton9 you can fill-in your prefered time for a kickstart there http://whenisgood.net/xbdpxd2/ and we can work on a regular scheduled meeting later.

**Comment by martindurant** [comment]
I'm not sure I can promise my full attention in the near future, please start without me.

**Comment by Carreau** [comment]
With the current responses it looks like the available times are mostly Wed 27th, 12 to 3pm US Pacific, 7-9 PM US East, 7-9pm GMT. Let's use the same link as community meeting for this week.

**Comment by Carreau** [comment]
We are in zoom.us/j/300670033

**Comment by joshmoore** [comment]
Today is again the [regular zarr/n5](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit#heading=h.mw5m9r245ow0) conference call. There's currently nothing on the agenda, but I would assume we'll hear something about last week's spec discussion. (I may be slightly late due to a graduation celebration if someone else wants to get things going)

**Comment by Carreau** [comment]
See #33 for v3 spec regular call.

**Comment by joshmoore** [comment]
It's that time again: Happy Community Call Wednesday, see the description for coordinates.

**Comment by joshmoore** [comment]
_Whoa!_ Two weeks gone already. See you at the community call.

**Comment by martindurant** [comment]
Today is Canada Day; but I can report that fsspec's mapper class has acquired the methods "getitems", "setitems" and "delitems", which will be implemented in async/concurrent when possible. This already works for HTTP (read-only), and I am doing GCS/S3 next.

**Comment by joshmoore** [comment]
Having missed the week before last, I'm already out of practice, so a bit late but: community meeting this afternoon if anyone has anything they'd like to discuss.

**Comment by rabernat** [comment]
I will show up for a bit and share our progress on [rechunker](https://rechunker.readthedocs.io/).

**Comment by joshmoore** [comment]
Happy (every other) Wednesday, everyone. If you happen to not be doing something more fun, feel free to drop by. (I'll be there if my providers' servers don't fully melt...)

**Comment by joshmoore** [comment]
Updated the title to be *super* clear. See (some of) you in a bit.

**Comment by jpivarski** [comment]
I see that the next meeting is on Wednesday, October 7th. Is the time still "8-9pm GMT" (and place still https://zoom.us/j/300670033) as stated in the first message? (Subsequent messages have talked about rescheduling and Doodle polls.) Thanks!

(Came here from https://github.com/zarr-developers/community/issues/33#issuecomment-702863760)

**Comment by jakirkham** [comment]
Yeah that's accurate. We've been editing the OP to stay up-to-date.

**Comment by jakirkham** [comment]
Sounds like it was not accurate. I've edited the OP to correct the time.

**Comment by joshmoore** [comment]
Updated the zoom link to include the passcode. Note: for other CET'ers, the time is now in UTC which means that it's currently at 2000 Berlin. @ryan-williams has a calendar invite that you can be added to to manage the time shifts.

**Comment by jakirkham** [comment]
@ryan-williams, I don't know if this is possible, but would we be able to include a link to the calendar event here and in the notes? If not, no worries.

**Comment by ryan-williams** [comment]
@jakirkham [here is a link](https://calendar.google.com/event?action=TEMPLATE&tmeid=dWJrMG83c3JrN2RiYzBpNW5oazFsZTRxZ3JfMjAyMDExMThUMTkwMDAwWiByeWFuLndpbGxpYW1zQG1zc20uZWR1&tmsrc=ryan.williams%40mssm.edu&scp=ALL) gcal gives me via a "Publish event" flow. It seems to be stable across a small tweak I just made to the event description to test it, so I think it works the way you'd expect.

I've updated the Zoom link in the cal event to include the pswd query param now as well.

@joshmoore reports separately that he's seeing it at an incorrect time on his calendar though, so something still might be off… please chime in if you see anything.

Here's how it looks for me:
![image](https://user-images.githubusercontent.com/465045/98861109-bd6e0180-2432-11eb-92e0-d90bfc725fad.png)
 (the guest list continues)

**Comment by jakirkham** [comment]
Thanks Ryan! 😄

When I click the link, Google Calendar opens, but it says "Could not find the requested event." Is there a different link maybe?

**Comment by ryan-williams** [comment]
that's frustrating. It loads for me in a Chrome profile logged in to google account that I didn't think was affiliated with the event at all, but sure enough from another google account I see what you are describing. I don't see another way to get a link to the event 😔 sorry, lmk if you have other suggestions. I'm noting this in my previous comment also.

**Comment by jakirkham** [comment]
No worries. Tinkered with an event in Google Calendar, but didn't come up with anything either unfortunately 😞

**Comment by joshmoore** [comment]
Just a heads up that we are still having timezone issues. Some people showed up at German 2100.

**Comment by d70-t** [comment]
Hay @joshmoore, thanks for having a chat this evening.

For the time issue, I think the simplest fix would be to replace `UTC` by `Europe/London` in the [introductory post](https://github.com/zarr-developers/community/issues/1#issue-463966208), however probably it would be even better to set the time and timezone to whatever it is actually scheduled in (such that different switch points for daylight saving time could be accounted for properly). For reference, here are a few times with conversions to various time zones using www.timeanddate.com:

| reference | UTC | Europe/London | Europe/Berlin | America/New_York | America/Los_Angeles |
|:--- |:--- |:--- |:--- |:--- |:--- |
| [19:00 UTC JAN](https://www.timeanddate.com/worldclock/converter.html?iso=20210120T190000&p1=1440&p2=136&p3=37&p4=179&p5=137) | 19:00 | 19:00 GMT | 20:00 CET | 14:00 EST | 11:00 PST |
| [19:00 Europe/London JAN](https://www.timeanddate.com/worldclock/converter.html?iso=20210120T190000&p1=1440&p2=136&p3=37&p4=179&p5=137) | 19:00 | 19:00 GMT | 20:00 CET | 14:00 EST | 11:00 PST |
| [19:00 UTC MAR](https://www.timeanddate.com/worldclock/converter.html?iso=20210320T190000&p1=1440&p2=136&p3=37&p4=179&p5=137) | 19:00 | 19:00 GMT | 20:00 CET | 15:00 EDT | 12:00 PDT |
| [19:00 Europe/London MAR](https://www.timeanddate.com/worldclock/converter.html?iso=20210320T190000&p1=1440&p2=136&p3=37&p4=179&p5=137) | 19:00 | 19:00 GMT | 20:00 CET | 15:00 EDT | 12:00 PDT |
| [19:00 UTC APR](https://www.timeanddate.com/worldclock/converter.html?iso=20210420T190000&p1=1440&p2=136&p3=37&p4=179&p5=137) | 19:00 | 20:00 BST | 21:00 CEST | 15:00 EDT | 12:00 PDT |
| [19:00 Europe/London APR](https://www.timeanddate.com/worldclock/converter.html?iso=20210420T180000&p1=1440&p2=136&p3=37&p4=179&p5=137) | 18:00 | 19:00 BST | 20:00 CEST | 14:00 EDT | 11:00 PDT |

So if local time should stay approximately constant, the time should be defined in any localized time zone. However, as [Daylight Saving Time changes at different days depending on country](https://en.wikipedia.org/wiki/Daylight_saving_time_by_country), it will not stay the same wall clock time for everyone at all times, i.e. 14:00 America/New_York is most of the time equal to 19:00 Europe/London, but sometimes (e.g. mid of March) not.

**Comment by ryan-williams** [comment]
Sorry that this continues to be a problem! Indeed, the calendar event currently says 6-7pm GMT ([screenshot](https://user-images.githubusercontent.com/465045/115631648-39b71b80-a2d4-11eb-8e67-c568b562fed8.png)) while the OP says 7-8pm UTC (meaning they're off by 1hr; GMT == UTC for our purposes here, afaik).

Since most participants are in locales with daylight savings, I like the suggestion to anchor it in someone's local time, that way it will stay at the same local time for everyone (except for the last 2wks of March and the first week of November, when US and Europe are an hour closer, iiuc).

**I propose changing to 7-8pm UK time**, which will keep it at the current calendar-event time for the next several months, will always be 7pm in the UK and 8pm for @joshmoore in Germany, and will almost always be 11am PT / 2pm ET in the US (except right around DST shifts).

Give me a 👍/👎 or lmk if I've made a mistake, and I'll update the cal event tmrw if it sg to all.

**Comment by d70-t** [comment]
Just one small addition: when referring to a local zone, I'd recommend to use Area/City based time zone codes from the [IANA timezone database file](https://data.iana.org/time-zones/tzdb/zone1970.tab) those have well defined semantics and are continuously maintained. In this case, it would be Europe/London in stead of UK.

---

Probably this is going to far, but I spent some time about time in the last weeks... It would also be interesting in which time zone the calender entry is set up under the hood. Usually, calendar tools have one time zone which sets the date of the event and another time zone which is the displayed time. As it can be seen in the table above, setting a recurring event in `GMT` (== `UTC` for these purposes) is different from setting an event in `BST` which again is different from setting an event in `Europe/London`. And setting an event in `Europe/London` and viewing at it in `GMT` is different from setting an event in `GMT` and viewing it in `GMT` 😬, even if they sometimes look the same.

**Comment by ryan-williams** [comment]
Thanks for all the info @d70-t! I've just updated the event. I was looking for something like "Europe/London" but the only option I saw in Google Calendar was "(GMT+01:00) United Kingdom Time":
![screenshot of google calendar time-zone picker](https://user-images.githubusercontent.com/465045/115722974-7f63fa80-a34d-11eb-9d81-2757ca72e840.png)

Hopefully it will be at a stable local time for most of us now going forward.

**Comment by jkh1** [comment]
Reading this thread I am still not sure of the time of the meeting. Is it 19:00 Europe/London or 20:00 Europe/London? The top two posts with tl;dr are not in agreement. Maybe putting the time of the next meeting at the top of the agenda file would help.

**Comment by joshmoore** [comment]
I've updated the second post to agree with the first.

**Comment by joshmoore** [comment]
Thanks to @DennisHeimbigner for presenting his [slides](https://github.com/zarr-developers/community/files/6429601/NCZarr.Filters.pdf) on filters in nczarr. Plan is to introduce a registry of V2 filters on the numcodecs repo. More details in the [notes](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit#) cc: @alimanfoo @cgohlke

A heads up: the next meeting (May 19th) overlaps with the Dask Summit (https://summit.dask.org/schedule) and is therefore cancelled. You can join several of us during [Social Event: Quiz Night](https://summit.dask.org/schedule/presentation/53/social-event-quiz-night/), for example, if you'd like.

**Comment by joshmoore** [comment]
With SciPy going on, it's unlikely anyone will be available for today's community call. Of course, the Zoom is open if anyone needs it. :wink:

**Comment by joshmoore** [comment]
All,

As discussed during the [last call](https://j.mp/zarr-community-1) (Dec 15), the Zarr calls are going into hiatus until the new year. We'll be back on the 12th of January (i.e. skipping December 29th). If you didn't see, there's also now a [Google calendar](https://calendar.google.com/calendar/embed?src=c_ba2k79i3u0lkf49vo0jre27j14%40group.calendar.google.com&ctz=Europe%2FBerlin) ([ics](https://calendar.google.com/calendar/ical/c_ba2k79i3u0lkf49vo0jre27j14%40group.calendar.google.com/public/basic.ics)) in case you want to subscribe.

Regardless, hope everyone has a few safe & restful weeks.
~Josh

**Comment by joshmoore** [comment]
All the info for the meetings is now under https://zarr.dev/#community-meetings :tada: closing.

**Comment by jakirkham** [comment]
Unpinned the issue too. Also submitted PR ( https://github.com/zarr-developers/community/pull/48 ) to add the meeting information to the README of this repo
